src/languages/en.ts

```diff
@@ -2234,7 +2231,16 @@ export default {
             accounts: 'Chart of accounts',
             taxes: 'Taxes',
             imported: 'Imported',
-            importedAsTags: 'Imported, displayed as tags',
+            notImported: 'Not imported',
+            importAsCategory: 'Imported, displayed as categories',
+            importTypes: {
+                [CONST.INTEGRATION_ENTITY_MAP_TYPES.IMPORTED]: 'Imported, displayed as tags',
+                [CONST.INTEGRATION_ENTITY_MAP_TYPES.TAG]: 'Imported, displayed as tags',
+                [CONST.INTEGRATION_ENTITY_MAP_TYPES.DEFAULT]: 'Imported, displayed as tags',
```

Comment:
Should this be `displayed as tags`? or `Xero default?`

Reply:
Oh, your are right,  I think before this was showing as `Imported`